crates/viewer/re_viewer_context/src/time_control.rs

```diff
@@ -42,6 +45,16 @@ trait TimeBlueprintExt {
 
     fn set_fps(&self, fps: f64);
     fn get_fps(&self) -> Option<f64>;
+
+    fn set_play_state(&self, play_state: PlayState);
+    fn get_play_state(&self) -> Option<PlayState>;
+
+    fn set_loop_mode(&self, loop_mode: LoopMode);
+    fn get_loop_mode(&self) -> Option<LoopMode>;
+
+    fn set_time_selection(&self, time_range: AbsoluteTimeRange);
+    fn get_time_selection(&self) -> Option<AbsoluteTimeRange>;
+    fn clear_time_selection(&self);
```

Comment:
you didn't start the fire here, but still would be nice to be idiomatic:

```suggestion
    fn set_play_state(&self, play_state: PlayState);
    fn play_state(&self) -> Option<PlayState>;

    fn set_loop_mode(&self, loop_mode: LoopMode);
    fn loop_mode(&self) -> Option<LoopMode>;

    fn set_time_selection(&self, time_range: AbsoluteTimeRange);
    fn time_selection(&self) -> Option<AbsoluteTimeRange>;
    fn clear_time_selection(&self);
```